dask/local.py

```diff
@@ -61,7 +61,7 @@
 Examples
 --------
 
->>> import pprint  # doctest: +SKIP
+>>> import pprint
```

Comment:
I'm not sure how much value there is in unskipping an import if the whole rest of the doctest is skipped. If seems like you could probably unskip these tests though.

Reply:
Done.